src/main/resources/avro/bdg.avdl

```diff
@@ -1221,4 +1221,166 @@ record Sample {
    */
   map<string> attributes = {};
 }
-}
\ No newline at end of file
+
+/**
+ Alphabet.
+ */
+enum Alphabet {
+
+  /**
+   DNA alphabet.
+   */
+  DNA,
```

Comment:
Currently I'm using it to map [to](https://github.com/heuermh/dishevelled-bio/blob/master/convert/src/main/java/org/dishevelled/bio/convert/biojava/BdgenomicsAlphabetToBiojavaAlphabet.java) and [from](https://github.com/heuermh/dishevelled-bio/blob/master/convert/src/main/java/org/dishevelled/bio/convert/biojava/BiojavaAlphabetToBdgenomicsAlphabet.java) Biojava [Alphabet](http://biojava.org/docs/api1.9.1/org/biojava/bio/symbol/Alphabet.html), which is based on IUPAC, but entirely extensible.  Here I see it as simply informational.  Unless of course we want to fully model sequences as symbol lists where symbols come from alphabets, which was discussed elsewhere.